src/mixxx.cpp

```diff
@@ -1508,9 +1508,12 @@ bool MixxxMainWindow::eventFilter(QObject* obj, QEvent* event) {
         // return true for no tool tips
         switch (m_toolTipsCfg) {
             case mixxx::TooltipsPreference::TOOLTIPS_ONLY_IN_LIBRARY:
-                return dynamic_cast<WBaseWidget*>(obj) != nullptr;
+                if (dynamic_cast<WBaseWidget*>(obj) != nullptr) {
```

Comment:
This can become a qobject_cast

Reply:
WBaseWidget is not a QObject but plain class used as mixin.

Reply:
Confirmed.